pages.de/common/cradle-elastic.md

```diff
@@ -0,0 +1,28 @@
+# cradle elastic
+
+> Management der ElasticSearch Instanzen einer Cradle Instanz.
+> Weitere Informationen: <https://cradlephp.github.io/docs/3.B.-Reference-Command-Line-Tools.html#elastic>.
+
+- Entleeren des ElasticSearch Indexes:
+
+`cradle elastic flush`
+
+- Entleeren des ElasticSearch Indexes für ein bestimmtes Paket:
+
+`cradle elastic flush {{package_name}}`
```

Comment:
Please translate the arguments for get this easier for non-English speakers in all pages.

Reply:
Good catch, of course 👍